### PR TITLE
Bunch of changes

### PR DESCRIPTION
**Pull request by TortillaChip**
Read the commits please

### CODE REVIEW

File: app/models/group_membership.rb [hunk 1]
```diff
@@ -1,5 +1,4 @@
 class GroupMembership < ApplicationRecord
-  belongs_to :language
```
Comment (TortillaChip) [inline app/models/group_membership.rb]: this becomes unnecessary because of the new schema

File: app/models/language.rb [hunk 2]
```diff
@@ -1,7 +1,6 @@
 class Language < ApplicationRecord
   has_many :card_sets
   has_many :groups
-  has_many :group_memberships
```
Comment (TortillaChip) [inline app/models/language.rb]: This as well

File: app/views/card_sets/_set_card_info_right.html.erb [hunk 3]
```diff
@@ -1,20 +1,24 @@
...
+      <h4><%= set.title %> - <%= set.difficulty %></h4>
+      <div class="dropdown-xp">
+        <p><%= set.flashcards.count * @difficulty_value %>XP</p>
       </div>
-      <h4 class="text-white">
-        <%= set.description %>
-      </h4>
+    </div>
+    <div class="dropdown-description">
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis massa arcu, tincidunt in lectus interdum, dignissim congue magna. Nullam imperdiet sagittis mauris eget semper. Nunc maximus quis risus id eleifend. </p>
```
Comment (rahulkeerthi) [inline app/views/card_sets/_set_card_info_right.html.erb]:
```suggestion
      <p><%= set.description %></p>
```

File: app/views/card_sets/_set_card_info_left.html.erb [hunk 4]
```diff
@@ -1,20 +1,24 @@
...
+      <h4><%= set.title %> - <%= set.difficulty %></h4>
+      <div class="dropdown-xp">
+        <p><%= set.flashcards.count * @difficulty_value %>XP</p>
       </div>
-      <h4 class="text-white">
-        <%= set.description %>
-      </h4>
+    </div>
+    <div class="dropdown-description">
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis massa arcu, tincidunt in lectus interdum, dignissim congue magna. Nullam imperdiet sagittis mauris eget semper. Nunc maximus quis risus id eleifend. </p>
```
Comment (rahulkeerthi) [inline app/views/card_sets/_set_card_info_left.html.erb]:
```suggestion
      <p><%= set.description %></p>
```

**Pull request merged**